TipulDay.py

```diff
@@ -1,178 +1,182 @@
+
+    # parse the HTML
     soup = BeautifulSoup(response.text, 'html.parser')
 
-        # get the top 1000 websites from the search results
+    # get the top 1000 websites from the search results
     for link in soup.find_all('a'):
         if link.get('href').startswith('/url?q='):
             websites.append(link.get('href')[7:])
-    
-    SearchStartNum=SearchStartNum+10 #increases after every loop to start in +10 search value
-    url=f'https://www.google.com/search?q={SearchString}&start={SearchStartNum}'
 
-print("Finished Collecting Website Data.")
+    # increases after every loop to start in +10 search value
+    SearchStartNum = SearchStartNum+10
 
-###############################################################################################################################################################################
+print("[*] Websites collected")
 
-## Code that inputs the form into all the websites
+###############################################################################################################################################################################
 
-print("Started Submitting Forms.")
+# Code that inputs the form into all the websites
 
-if(len(websites)>0):
+print("[*] Submitting forms")
 
-    SuccessfullForms=0
-    FailedForms=0
+if(len(websites) > 0):
+    SuccessfullForms = 0
+    FailedForms = 0
     for website in websites:
-        #specifying the website url
+        # specifying the website url
         url = website
 
-        #making a request to the website and getting the response 
+        # making a request to the website and getting the response
         try:
             response = requests.get(url)
         except:
-            print("Error 5: Issue requesting from the website.")
-            FailedForms=FailedForms+1
+            print("[!] Error 5: Issue requesting from the website")
+            FailedForms += 1
             continue
 
-        #parsing the response using BeautifulSoup
+        # parsing the response using BeautifulSoup
         soup = BeautifulSoup(response.text, 'html.parser')
 
-        #finding the form elements in the page 
+        # finding the form elements in the page
         form_elements = soup.find_all('input')
         #form_elements.append(soup.find_all('input', {'type': 'tel'}))
-        #creating a dictionary to store the form data
+        # creating a dictionary to store the form data
         form_data = {}
 
-        #looping through the form elements and getting the data 
+        # looping through the form elements and getting the data
         for element in form_elements:
-                name = element.get('name')
-                value = element.get('value')
-                form_data[name] = value
+            name = element.get('name')
+            value = element.get('value')  # why is the value necessary?
+            form_data[name] = value
 
-    
         for data in form_data:
-                    DataName =data
```

Comment:
for data in form_data:
                    DataName =data
                    try:
                        if "name" in DataName or "fname" in DataName or "lname" in DataName or "1" in DataName or "form_field_1" in DataName or "contact_name" in DataName:
                                form_data[DataName] = FormNameValue
                        if "phone" in DataName or "Phonenumber" in DataName or "Phonenum" in DataName or "num" in DataName or "tel" in DataName or "telephone" in DataName or "2" in DataName or "form_field_2" in DataName or "contact_phone" in DataName:
                                form_data[DataName] = FormPhoneNumberValue  
                        if "mail" in DataName or "email" in DataName or "3" in DataName or "form_field_3" in DataName or "contact_email" in DataName :
                                form_data[DataName] = FormEmailValue   
                        if "Comment" in DataName or "message" in DataName or "details" in DataName  or "text" in DataName or "4" in DataName or "form_field_4" in DataName  or "contact_message" in DataName : 
                                form_data[DataName] = FormCommentValue  
                    except:
                        print('Error 6: Some parameters might be missing:')


 NOTE TO MYSELF TO UPDATE